### PR TITLE
update terraform integration documentation

### PR DESCRIPTION
**Pull request by pawelsz-rb**
## Description of the change

Update integration documentation

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Maintenance
- [ ] New release

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers assigned
- [ ] Issue from task tracker has a link to this pull request
- [ ] Changes have been reviewed by at least one other engineer

### CODE REVIEW

File: docs/resources/integration.md [hunk 1]
```diff
@@ -34,6 +37,36 @@ resource "rollbar_integration" "slack_integration" {
...
   }
 }
+
+# Configure the Webhook integration for the project
+#
+
+resource "rollbar_integration" "webhook_integration" {
+  webhook {
+    enabled = true
+    url = "https://www.some-website.com"
```
Comment (rollbar-bborsits) [inline docs/resources/integration.md:47]: some-website.com is a real URL. Can you change it to example.com?

Comment (pawelsz-rb) [reply docs/resources/integration.md]: done

File: docs/resources/integration.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
 `rollbar_integration` Resource
 =========================
 
-Rollbar projects can be configured with different notification integrations (aka "channels") and rules for when to send notifications to those integration platforms.  This resource manages the configuration for the integrations(limited to the Slack channel at the moment) for the project configured for the Rollbar provider.
+Rollbar projects can be configured with different notification integrations (aka "channels") and rules for when to send notifications to those integration platforms.  This resource manages the configuration for the integrations(limited to the Slack, Webhook, Email and PagerDuty channels at the moment) for the project configured for the Rollbar provider.
 
-This resource can manage configuration for the Slack channel. See the following api documentation for more details about the arguments with respect to the Slack channel:
+This resource can manage configuration for the Slack, Webhook, Email and PagerDuty channels. See the following api documentation for more details about the arguments with respect to these channels:
```
[the remark this answers is not in the document]
Comment (pawelsz-rb) [reply docs/resources/integration.md:6]: done

**Review by rollbar-bborsits — APPROVED** [review]
LGTM, thanks

**Pull request merged**